### PR TITLE
Regenerated using latest generator

### PR DESCRIPTION
**Pull request by Guite**
| Q                 | A
| ----------------- | ---
| Bug fix?          | yes
| New feature?      | no
| BC breaks?        | no
| Deprecations?     | no
| Fixed tickets     | -
| Refs tickets      | -
| License           | MIT
| Changelog updated | no

### CODE REVIEW

File: src/system/RoutesModule/Entity/Base/AbstractRouteEntity.php [hunk 1]
```diff
@@ -685,6 +686,8 @@ public function setGroup($group)
         $this->group = isset($group) ? $group : '';
     }
     
+    
+
```
Comment (craigh) [inline src/system/RoutesModule/Entity/Base/AbstractRouteEntity.php]: ?

Comment (Guite) [inline src/system/RoutesModule/Entity/Base/AbstractRouteEntity.php]: nvm for now

File: src/system/RoutesModule/Entity/Base/AbstractRouteEntity.php [hunk 2]
```diff
@@ -701,7 +704,7 @@ public function getTitleFromDisplayPattern()
     
         return $formattedTitle;
     }
-
+
```
Comment (craigh) [inline src/system/RoutesModule/Entity/Base/AbstractRouteEntity.php]: ?

File: src/system/RoutesModule/Twig/Base/AbstractTwigExtension.php [hunk 3]
```diff
@@ -219,7 +219,7 @@ public function getUserAvatar($uid = 0, $width = 0, $height = 0, $size = 0, $rat
             if (!count($results)) {
                 return '';
             }
-
+    
             $uid = $results->getIterator()->getArrayCopy()[0]->getUname();
```
Comment (craigh) [inline src/system/RoutesModule/Twig/Base/AbstractTwigExtension.php]: ?

File: src/system/RoutesModule/Entity/Base/AbstractRouteEntity.php (hunk 2 above)
Comment (Guite) [inline src/system/RoutesModule/Entity/Base/AbstractRouteEntity.php]: :roll_eyes:

File: src/system/RoutesModule/Twig/Base/AbstractTwigExtension.php (hunk 3 above)
Comment (Guite) [inline src/system/RoutesModule/Twig/Base/AbstractTwigExtension.php]: You know it already: https://bugs.eclipse.org/bugs/show_bug.cgi?id=398172

Comment (craigh) [inline src/system/RoutesModule/Twig/Base/AbstractTwigExtension.php]: 😞

**Pull request merged**